tests/029.solrdocument_serialize_php81.phpt

```diff
@@ -0,0 +1,50 @@
+--TEST--
```

Comment:
Consider not to split the tests; it might be okay to use some placeholders for the version-dependent parts (if all else fails, you can use regular expressions inside of `%r` markers, e.g. `%rC:20:"SolrModifiableParams"|O:12:"SolrDocument"%r`).

Reply:
I kinda like having the tests split because I am going with the different api for 8.1+.
Also, the difference in the serialization output is quite different between the old and the new.
```
C:12:"SolrDocument":172:{<?xml version="1.0" encoding="UTF-8"?>
```
vs.
```
O:12:"SolrDocument":1:{s:3:"xml";s:172:"<?xml version="1.0" encoding="UTF-8"?>
```

And then also the ending.